### PR TITLE
Removes MicahZoltu as EIP editor.

### PR DESCRIPTION
**Pull request by MicahZoltu**
**Context**: We now have other editors who are relatively active so the response time for authors isn't as bad as it used to be (a week vs a year).  It also sounds like the process for consensus changes will be moving over to the executable specs repository in the near future and since that is the piece I care most about, I am less concerned now than I used to be about the EIP process remaining credibly neutral.

I also will be unsubscribing from this repository, so someone should probably take up the the task of making sure stuff doesn't get lost in the cracks.

**Note**: I also reordered the Emeritus editors list to be alphabetical rather than what appeared to be random order.

### CODE REVIEW

File: .github/workflows/auto-merge-bot.yml
Comment (Pandapip1) [inline .github/workflows/auto-merge-bot.yml:25]:
```suggestion
          INFORMATIONAL_EDITORS: "@MicahZoltu,@lightclient,@axic,@gcolvin,@SamWilsn,@Pandapip1"
```
I feel like this particular change is very important.

**Review by Pandapip1 — CHANGES_REQUESTED** [review]
In all seriousness:
 - Someone else will need to be granted admin for this repository. There are enough changes that need to be manually merged, so having an active admin is essential. I propose @SamWilsn for this.
 - You are currently the only active maintainer of the EIP-Bot repository. I feel pretty comfortable with the codebase there, and would be happy to help maintain it. But that means that at the very least I need write access (and preferably admin).

**Comment by MicahZoltu** [comment]
I agree, that the repository will need someone with write permission who is active.  I second your nomination of Sam, but I believe Sam explicitly said they don't want it in the past.

**Comment by SamWilsn** [comment]
If you're leaving, @MicahZoltu, I'll accept write permissions.

**Comment by MicahZoltu** [comment]
I don't have permission to give you write permissions, @poojaranjan knows how to contact the right people for that though.

**Comment by Pandapip1** [comment]
If https://github.com/ethereum/EIPs/pull/5422 is implemented this will be a lot simpler.

**Comment by poojaranjan** [comment]
> I don't have permission to give you write permissions, @poojaranjan knows how to contact the right people for that though. Need Sam added to this team: https://github.com/orgs/ethereum/teams/eip-editors

I have requested the team to do the needful.

**Comment by MicahZoltu** [comment]
As my first, latest, and final act as Dictator of the EIP Editor process, I'm going to abuse my power and merge this without waiting for approval.  😈 

Here is a picture of an innocent kitten to manipulate the emotional context that this event is stored with in your brain:
image

**Pull request merged**
